### PR TITLE
Clarifying deprecated item of v1 without a group

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://deploy-preview-31775--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-deprecated-removed-features

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Note to self, if this is approved, update 4.8 RNs as well.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -998,7 +998,7 @@ In the table, features are marked with the following statuses:
 |GA
 |DEP
 
-|Use of `v1` in `apiVersion` for {product-title} resources
+|Use of `v1` without a group in `apiVersion` for {product-title} resources
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:1001]: @soltysh Can you let me know if this clarification is appropriate?

Preview: https://deploy-preview-31775--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-deprecated-removed-features

When I read it the first time it confused me into thinking it was all v1 references, even if there was a group in front of it.

Comment (soltysh) [reply release_notes/ocp-4-7-release-notes.adoc:1001]: Yeah, this is good, thx!

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request closed without merging**